### PR TITLE
Updated release notes to add the software supply chain security elements

### PR DESCRIPTION
**Pull request by gtrivedi88**
Version(s):
please cp to pipelines-docs-1.14

Issue: [HACDOCS-845](https://issues.redhat.com/browse/HACDOCS-845)

Link to docs preview:

SME Review (completed): @karthikjeeyar 
QE review (completed): @ppitonak 
CS review: @Preeticp 
DPM review:
PM review: @koustavsaha
Product experience review:
Peer-review:

### CODE REVIEW

File: modules/op-release-notes-1-14.adoc [hunk 1]
```diff
@@ -171,6 +171,10 @@ To use the web console plugin for {pipelines-shortname}, you must use, at a mini
 
 * With this update, if you are using {OCP} 4.15 and you enabled the console plugin, you can view archive information about past pipeline runs and task runs. {tekton-results} provides this information.
 
+* With this update, `PipelineRun` details page, accessible from both the *Developer* or *Administrator* perspective of the web console, introduces a *Vulnerabilities* row. This new row offers a visual representation of identified vulnerabilities, categorized by the severity (critical, high, medium, and low). To enable this feature, update your tasks and associated pipelines to the specified format. Additionally, once enabled, you can access the *Vulnerabilities* information through the Vulnerabilities column in the pipeline run list view page.
```
Comment (mramendi) [inline modules/op-release-notes-1-14.adoc:174]:
```suggestion
* With this update, the `PipelineRun` details page, accessible from both the *Developer* or *Administrator* perspective of the web console, introduces a *Vulnerabilities* row. This new row offers a visual representation of identified vulnerabilities, categorized by the severity (critical, high, medium, and low). To enable this feature, update your tasks and associated pipelines to the specified format. Additionally, once enabled, you can access the information about identified vulnerabilities through the *Vulnerabilities* column in the pipeline run list view page.
```

File: modules/op-release-notes-1-14.adoc [hunk 2]
```diff
@@ -171,6 +171,10 @@ To use the web console plugin for {pipelines-shortname}, you must use, at a mini
...
+
+* With this update, `PipelineRun` details page, accessible from both the *Developer* or *Administrator* perspective of the web console, provides an option to download or view Software Bill of Materials (SBOMs) for enhanced transparency and control. To enable this feature, update your tasks and associated pipelines to the specified format.
```
Comment (mramendi) [inline modules/op-release-notes-1-14.adoc:176]: Add "the" here too

**Review by mramendi — APPROVED** [review]

File: modules/op-release-notes-1-14.adoc [hunk 3]
```diff
@@ -171,6 +171,10 @@ To use the web console plugin for {pipelines-shortname}, you must use, at a mini
 
 * With this update, if you are using {OCP} 4.15 and you enabled the console plugin, you can view archive information about past pipeline runs and task runs. {tekton-results} provides this information.
 
+* With this update, the `PipelineRun` details page, accessible from both the *Developer* or *Administrator* perspective of the web console, introduces a *Vulnerabilities* row. This new row offers a visual representation of identified vulnerabilities, categorized by the severity (critical, high, medium, and low). To enable this feature, update your tasks and associated pipelines to the specified format. Additionally, once enabled, you can also access the information about identified vulnerabilities through the *Vulnerabilities* column in the pipeline run list view page.
```
Comment (karthikjeeyar) [inline modules/op-release-notes-1-14.adoc:174]:
```suggestion
* With this update, the `PipelineRun` details page, accessible from both the *Developer* or *Administrator* perspective of the web console, introduces a *Vulnerabilities* column. This new column offers a visual representation of identified vulnerabilities, categorized by the severity (critical, high, medium, and low). To enable this feature, update your tasks and associated pipelines to the specified format. Additionally, once enabled, you can also access the information about identified vulnerabilities through the *Vulnerabilities* column in the pipeline run list view page.
```

**Review by karthikjeeyar — APPROVED** [review]
LGTM

**Review by ppitonak — APPROVED** [review]

**Review by kowen-rh — COMMENTED** [review]
No documentation preview was linked, so my review is based entirely on the content of the diff.

/remove-label peer-review-in-progress
/label peer-review-done

File: modules/op-release-notes-1-14.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (mramendi) [reply modules/op-release-notes-1-14.adoc:174]: There are separate list and detail views for "pipeline runs". A pipeline run is a particular instance/case of running the pipeline. One pipeline can be run many times and each time is a separate "pipeline run", and there is a PipelineRun constom resource to represent it

[the remark this answers is not in the document]
Comment (kowen-rh) [reply modules/op-release-notes-1-14.adoc:174]: Ah, that makes sense. Thanks for the explanation!

**Comment by mramendi** [comment]
The doc preview is at https://72600--ocpdocs-pr.netlify.app/openshift-pipelines/latest/about/op-release-notes#new-features-1-14_op-release-notes under Web Console and I have now checked it.

**Comment by mburke5678** [comment]
/lgtm

@gtrivedi88 Are we waiting on CS, DPM, PM, and PX reviews?

**Comment by mramendi** [comment]
@mburke5678 no - the RNs for Pipelines require just SME and QE, others are optional

**Pull request merged**